lib/enigma.rb

```diff
@@ -1,20 +1,18 @@
 require './lib/cipher'
 
 class Enigma < Cipher
-  def encrypt(message, key, date)
+  def encrypt(message, key = generate_key, date = @date)
```

Comment:
This method was updated so that the key is generated if one isn't passed in as a method argument